recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -14,5 +14,10 @@ conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 :: KLUDGE to work around changes in conda-build 2.0.0
 conda install -n root --yes --quiet conda-build=1
```

Comment:
The conflict occurs because we have dropped this pinning to go to `conda-build` 2. Should be a trivial matter to fix this though.